RotationSolver/Updaters/TargetUpdater.cs

```diff
@@ -521,4 +524,28 @@ private static void UpdateNamePlate(IEnumerable<BattleChara> allTargets)
         }
         DataCenter.TreasureCharas = charas.ToArray();
     }
+
+    // status checks that can't easily be represented in config
+    private static bool CheckTargetableOther(BattleChara b) {
+        // Labyrinth of the Ancients
+        if (Svc.ClientState.TerritoryType == 174 && b.Name.ToString() == "Thanatos") {
+            // thanatos can only be damaged by players with Astral Realignment (398)
+            return Player.Status.Any(s => s.StatusId == 398);
+            // TODO: Allagan Bomb is also invulnerable until everything else is dead, but that's harder to check
+        }
+
+        // The Puppets' Bunker
+        if (Svc.ClientState.TerritoryType == 917) {
```

Comment:
Maybe create a new enum for different TerritoryTypes so we can avoid magic numbers in the logic, similar to `StatusID` or `ActionID`

Reply:
I don't expect to need to refer to territories by ID anywhere except this function so extracting it to an enum would qualify as bloat IMO